Second section projects and aboutself

Koshao-sys opened the pull request:
In this section i have worked on the following items;
1. Created the Projects section using CSS grid and flexboxes
2. Created the About Self section
3. Cleared al lint errors

Ibizugbe commented on index.html:
```diff
@@ -48,5 +48,156 @@ <h1 class="headline-job">I'm a software developer</h1>
         </div>
     </section>
+    <button class="btn-disabled"><img src="./Images//Disabled.svg" alt=""></button>
+    <section>
+        <h1 class="head-recent">My Recent <br> Works</h1>
+        <span class="recent-bar"></span>
```
Your project looks a little bit different from the given design, kindly fix it🤗

your project
<img width="263" alt="yourAppearance" src="https://user-images.githubusercontent.com/64643995/212398901-65772619-cde2-4436-83c8-720f5f622cf4.png">

the design
<img width="281" alt="figmaAppearance" src="https://user-images.githubusercontent.com/64643995/212398930-6ad78f94-8748-4062-a0ff-f7f2869c5342.png">

kindly make proper adjustments to the `spacing ` `<font-size>` and `<font-weight>`

Ibizugbe commented on index.html:
```diff
@@ -48,5 +48,156 @@ <h1 class="headline-job">I'm a software developer</h1>
+                          <button>SEE PROJECT</button>
+                      </div>
+                </div>
+            </div>
```
I think your project cards are looking slightly different from the design😉
your project
<img width="266" alt="projects-app" src="https://user-images.githubusercontent.com/64643995/212400825-fd73cbd4-05fd-48ba-b092-6dda68ab6154.png">
figma
<img width="275" alt="figmaProject" src="https://user-images.githubusercontent.com/64643995/212400886-769abd73-d353-4cc3-9f0c-b3c9dc560a54.png">

kindly take note of the spacings and the border and make the required changes🤗

Ibizugbe commented on index.html:
```diff
@@ -48,5 +48,156 @@ <h1 class="headline-job">I'm a software developer</h1>
+                              <li><a href="#">javScript</a></li>
+                              <li><a href="#">html</a></li>
+                          </ul>
+                          <button>SEE PROJECT</button>
```
I think your buttons for each projects are not having the required state as given by the design, kindly fix this🤗😉
<img width="247" alt="button states" src="https://user-images.githubusercontent.com/64643995/212406134-26c40eec-42ef-4916-ba2c-a41b42548bdf.png">

Ibizugbe commented on index.html:
```diff
@@ -48,5 +48,156 @@ <h1 class="headline-job">I'm a software developer</h1>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="about-myself">
+            <div>
+                <h2>About me</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <span><a href="#">Get My Resume</a></span>
+            </div>
```
Your about section looks a little bit different from the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=WZMqc3ufIF1I5Dt6-0). 

your app
<img width="278" alt="yourAbout" src="https://user-images.githubusercontent.com/64643995/212410689-94e7f795-14a6-4145-8474-21da607843b7.png">

the design
<img width="289" alt="designAbt" src="https://user-images.githubusercontent.com/64643995/212410745-42f2cde1-4ebb-43e0-83f8-e080521ab217.png">

kindly make the necessary changes. consider the background image, font properties, and spacing.🤗

Ibizugbe reviewed (changes requested):
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- [x] Linters are passing✅
- [x] the right github flow method has been used.✅

## Required Changes ♻️
- I suggest you complete all incomplete statements in your readme file, it will give your project a more professional look🤗
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Koshao-sys commented:
In this pull request I have worked on below items;
1. Worked on the general appearance to be identical
2. Worked on the hover and active features on the buttons.
3. Adding the background in the second section.

roshan-bajgain commented on index.html:
```diff
@@ -40,13 +40,165 @@ <h1 class="headline-job">I'm a software developer</h1>
+    <section>
+        <h1 class="head-recent">My Recent <br> Works</h1>
+        <span class="recent-bar"></span>
+        <div class="grid-container">
+            <div class="grid-item">
+              <div class="flex-container">
+                    <div class="works-image"></div>
+                    <div class="works-inner">
+                        <p>Multi-Post Stories<br>Gain+Glory</p>
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">css</a></li>
+                            <li><a href="#">javScript</a></li>
+                            <li><a href="#">html</a></li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
```
- [ ] As requested by the previous reviewer kindly make changes to the spacing and border. Use this border `1px solid #D0D9D4;` to match the Fimga design
- [ ] Kindly implement in all the cards as requested by the previous reviewer.
### Your design
![your](https://user-images.githubusercontent.com/81727988/212473448-9666793a-0f1b-4b7d-81e4-07ee2bba319b.jpg)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/212473452-025e7b12-8c18-4ff7-ac6c-2302c6b12e08.jpg)

roshan-bajgain commented on index.html:
```diff
@@ -40,13 +40,165 @@ <h1 class="headline-job">I'm a software developer</h1>
+                              <li><a href="#">css</a></li>
+                              <li><a href="#">javScript</a></li>
+                              <li><a href="#">html</a></li>
+                          </ul>
+                          <button>See Project</button>
+                      </div>
```
- [ ] As requested by the previous reviewer please implement the hover effect on your buttons. When I pressed the buttons then the effect did not work as in the Figma design
- [ ] Please implement in all buttons
### Design
![fimga](https://user-images.githubusercontent.com/81727988/212473628-09b8b762-d3f6-448d-9074-f33514b8d2dd.jpg)

roshan-bajgain reviewed (changes requested):
Hi @Koshao-sys, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
- [ ] Please remove the second author in your README file as you did this project alone as this is mentioned by the previous reviewer to updated README file.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Koshao-sys commented:
I have further made the changes below;
1. Removed the second Author from my README file.
2. I made adjustments to the cards in the works section.
3. Implemented the hover effect on all buttons.

ridwanediallo reviewed (approved):
# STATUS: APPROVED 🟢
## Hi @Koshao-sys , 
![Congratulations!GIF](https://user-images.githubusercontent.com/55918904/191301390-43d62dfa-bfdc-49c5-943b-4d72f4ea5bb8.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

- [OPTIONAL] In your readme file the `LICENSE` is missing please add it because it is one of the important things to have in your readme file. 🙂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
